sdcm/cluster.py

```diff
@@ -1639,7 +1637,6 @@ def node_setup(self, node, verbose=False, timeout=3600):
 
         node.wait_db_up(timeout=timeout)
         node.wait_jmx_up()
-        self.clean_replacement_node_ip(node, seed_address, endpoint_snitch)
 
     def clean_replacement_node_ip(self, node, seed_address, endpoint_snitch):
```

Comment:
What about `clean_replacement_node_ip` definition? I think we can remove it

Reply:
I agree. but @roydahan thinks it is beneficial to leave it there